src/Rules.UnitTests/LocalRuleMetadataProvider.RuleInfoDeserializerTests.cs

```diff
@@ -0,0 +1,80 @@
+  'Tags': [],
+  'DescriptionSections': [
+    {
+      'Key': 'introduction',
+      'HtmlContent': 'content1',
+      'Context': null
+    },
+    {
+      'Key': 'root_cause',
+      'HtmlContent': 'content2',
+      'Context': 
+        {
+            'Key': 'context key',
+            'DisplayName': 'context display name'
+        }
+    }
+  ],
+  'EducationPrinciples': [ 'aaa', 'bbb' ],
```

Comment:
The JS/TS analyzer doesn't seem to populate the `Context` or `EducationPrinciples` fields yet, but I added some test data to make the deserialization works.